operators/Makefile

```diff
@@ -361,3 +361,46 @@ check-license-header:
 check-local-changes:
 	@ [[ "$$(git status --porcelain)" == "" ]] \
 		|| ( echo -e "\nError: dirty local changes"; git status --porcelain; exit 1 )
+
+#########################
```

Comment:
Is there a way we could make this close to the way we use gke and minikube? I would expect something like
```
make bootstrap-kind    # starts a kind cluster through kind.sh
make set-context-kind # changes kubectl context to point to the kind cluster (called as part of bootstrap-kind)
make-e2e-local             # run e2e tests against the current k8s cluster
```

Reply:
I would be 👍 on adding `make set-context-kind`, unfortunately `Kind` does not seem able to update `$HOME/.kube/config`
Instead [it creates a config file for every Kind clusters](https://github.com/kubernetes-sigs/kind/blob/21af1750d1ce3723f6a92afda1391fb27d0e3747/site/content/docs/user/quick-start.md#interacting-with-your-cluster), we can't do it in the `Makefile` because a process can't change the environment of its parent, it's up to the user to export the variable.

Regarding `bootstrap-kind` I'm not sure about the default topology, I think that most of our e2e tests should be able to run on a single node cluster in order to make it easy/quick for developers to run them on their workstation before submitting a PR. On the other hand we still need 3 nodes cluster when it makes sense...

Reply:
Should we copy `kind` config into `~/.kube/config` ourselves as part of the `set-context-kind` target?
Something like:

```
KUBECONFIG=~/.kube/config:~/someotherconfig 
kubectl config view --flatten > ~/.kube/config
```

Reply:
For bootstrapping I think I'd be fine with running 1 node by default and being able to override it through an env var?

Reply:
> Should we copy `kind` config into `~/.kube/config` ourselves as part of the `set-context-kind` target?
> Something like:
> 
> ```
> KUBECONFIG=~/.kube/config:~/someotherconfig 
> kubectl config view --flatten > ~/.kube/config
> ```

It would erase the current configuration, it can be recreated but could be painful and as a user of [`kubectx` and `kubens`](https://github.com/ahmetb/kubectx) I'm not sure I would be happy if someone do that :)
I will check if it makes sense to contribute to kind to add an option that would update `~/.kube/config`